yalla-coop/connect5
# Forget password

relates to #318

to test it :
create an account 
- go to  ``/forget-password`` and input your gmail account 
- check your gmail and click the link then reset your password
- add in env file
``
EMAIL = your email(another email)
``
``
EMAIL_PASSWORD =  password
``

``
DOMAIN  = http://localhost:----
``
